Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -366,29 +370,30 @@ open class ChatMessageContentView: _View, ThemeProvider, UITextViewDelegate {
         }
 
         // Metadata
-        if options.hasFootnoteOptions {
```

Comment:
This is not checked anymore?

Reply:
This is not needed. We now check if we have footnote views, if yes, we render the footnote. The reason is that, the translation label is shown, independent of the lastInSequence logic